swarm/api/http/error_templates.go

```diff
@@ -129,10 +121,54 @@ func GetGenericErrorPage() string {
       }
 
     </style>
+`
+	return css
```

Comment:
I think this string (and many of the others in this file) should just be constants, for example:

```go
const templateCss = `
<style>
...
</style>
`
```

If we find we have many of these strings, we should consider using [go-bindata](https://github.com/jteeuwen/go-bindata) (like [the dashboard does](https://github.com/ethereum/go-ethereum/blob/master/dashboard/dashboard.go#L17-L23)) so that we can store these strings in separate files (e.g. `assets/css/style.css`), but still have them compiled into the binary.

More a passing comment than something I think needs addressing before merging this PR :+1:.

Reply:
yes it looks like we may need `go-bindata` for all these templates too